app/addons/documents/designdocinfo/components/DesignDocInfo.js

```diff
@@ -56,8 +56,9 @@ export default class DesignDocInfo extends React.Component {
       return <LoadLines />;
     }
     const viewIndex = this.props.viewIndex;
-    const actualSize = (viewIndex.data_size) ? viewIndex.data_size.toLocaleString('en') : 0;
-    const dataSize = (viewIndex.disk_size) ? viewIndex.disk_size.toLocaleString('en') : 0;
+    const {sizes} = viewIndex;
+    const actualSize = (sizes.active) ? sizes.active.toLocaleString('en') : 0;
+    const dataSize = (sizes.file) ? sizes.file.toLocaleString('en') : 0;
```

Comment:
The equivalent here is `sizes.external` according to https://github.com/apache/couchdb/pull/2163/files#diff-b8437e2de7ccbb213a215be755cb6935